### PR TITLE
Loads base templates instead of retrieving from API.

### PR DESCRIPTION
**Pull request by justinlittman**
part of #3266

## Why was this change made?
Easier maintenance and deploy of base templates.

### CODE REVIEW

File: src/sinopiaApi.js [hunk 1]
```diff
@@ -33,8 +50,12 @@ export const fetchResource = (
     } catch (err) {
       fetchPromise = Promise.reject(err)
     }
+  } else if (isBaseTemplateUri(uri)) {
+    fetchPromise = loadBaseTemplate(uri)
   } else if (Config.useResourceTemplateFixtures && isTemplate) {
     fetchPromise = Promise.reject(new Error("Not found"))
+  } else if (isBaseTemplateUri(uri)) {
+    fetchPromise = loadBaseTemplate(uri)
```
Comment (mjgiarlo) [inline src/sinopiaApi.js:58]: Repeats what's on lines 53-4. Intended or cruft?

Comment (justinlittman) [reply src/sinopiaApi.js:58]: Cruft. Removing.

**Review by mjgiarlo — APPROVED** [review]

**Review by ndushay — CHANGES_REQUESTED** [review]
I don't know if you're planning on multiple PRs -- eventually we want:

- changes to docker-compose.yml (?  or is this just in sinopia_api / mongo docker image?)
- changes to README

and obviously, in sinopia_api:
- removal of "base templates"
- README
- docker-compose 
- package.json dependencies removed??

Not sure about further changes related to deployment or what have you.

**Review by ndushay — CHANGES_REQUESTED** [review]
Closes #3266, no?   Typo in your description???

**Comment by justinlittman** [comment]
@ndushay I fixed the closes ref to #3266.

No need for changes to `docker-compose.yml`. This is all handled by https://github.com/LD4P/sinopia_api/pull/217.

Description of base templates in README would be useful, but beyond scope for this ticket.

**Review by ndushay — CHANGES_REQUESTED** [review]
any more testing "fixtures" that can go away?  Or maybe that's for separate ticketing.

**Review by ndushay — APPROVED** [review]
Approving, but changed so it doesn't close #3266 as that should get closed after sinopia_api changes and some documentation (which I started at https://github.com/LD4P/sinopia_editor/blob/editor-component-templates-doc/static/EditorComponentTemplates.md)

**Comment by justinlittman** [comment]
@ndushay Previously there was a single template fixture that was used as a dummy for all of the base templates. It has been removed.

**Comment by ndushay** [comment]
approved;  updated 3266 with the task list I envisioin.

**Pull request merged**